sections/webhooks.md

```diff
@@ -0,0 +1,138 @@
+# Webhooks (New!)
```

Comment:
❓ Is it possible to just say that each webhook will receive a copy of the object in the _same_ format you would, had you invoked a GET action with an id on the corresponding REST endpoint? I think this should be the case, and if we can confirm that, it could simplify the docs a bit?

Reply:
This is what it says, in the section on Webhook Payloads:
> The data contained in this POST is exactly the same as what would be returned if the actual object
were fetched through the API via a GET call.

This documentation is primarily on how to *register* a webhook. The payloads section simply directs them to the appropriate API pages for those objects.